x/emissions/keeper/inference_synthesis/forecast_implied.go

```diff
@@ -84,8 +88,12 @@ func CalcForecastImpliedInferences(args CalcForecastImpliedInferencesArgs) (
 				}
 
 				forecastImpliedInference := emissionstypes.Inference{
-					Inferer: forecaster,
-					Value:   medianValue,
+					TopicId:     args.TopicId,
```

Comment:
In this case, I understand we are adding fields that are not used(they were not previously), am I right? 
Would it make sense to use nolint here? 
I worry it may make data grow even bigger. 
Also if those other fields are not used, makes me think if `Inference` is the right structure to use here.

Reply:
I do not endorse using nolint because it means leaving those values as zero or nil, when they should not be
here for example the topic id is indeed the args.TopicId, regardless of whether it's used

and you are correct, Inference is not the right structure to use here: PROTO-2655